relay-server/src/services/server.rs

```diff
@@ -45,6 +46,13 @@ const KEEPALIVE_RETRIES: u32 = 5;
 /// header within this time, the connection is closed.
 const CLIENT_HEADER_TIMEOUT: Duration = Duration::from_secs(5);
 
+/// Only compress responses above this configured size.
```

Comment:
Could we add the unit to the docstrings? (assuming it's bytes)
```suggestion
/// Only compress responses above this configured size, in bytes.
```